### ISSUE TITLE
make it easier to see who wrote what

### ISSUE THREAD

**Issue by theoneandonly-vector** [nextcloud/talk-android#1526]
**Is your feature request related to a problem? Please describe.**
it's hard to follow a conversation when you can't see who wrote what. all parties write on the same side (left) and they are not differentiated by a name or color.

**Describe the solution you'd like**
Make it like the Desktop verison where you can see the name when someone writes. other options may be good too.

**Describe alternatives you've considered**
changing the theme doesn't change anything here.

**Comment by AndyScherzinger** [comment]
@theoneandonly-vector which desktop app are you refering to? There is the web UI which works the same way and the Android app also shows the avatar+name.

**Comment by theoneandonly-vector** [comment]
@AndyScherzinger hey there, yes I refer to the webUI. and nope at least on the current playstore version there you cannot see who wrote inside a chat with only two people. for groups it seems to work.

**Comment by AndyScherzinger** [comment]
Ah, that you mean. But what do you need for two people? You on the right, other person on the left. Name in the top bar including avatar, so, no need to show name and avatar all the time, no?

**Comment by theoneandonly-vector** [comment]
that would be true.. but somehow both write on the left side.

**Comment by theoneandonly-vector** [comment]
![Screenshot_20210813-195604](https://user-images.githubusercontent.com/10380051/129400003-4f7376f9-d646-491f-8c26-be49f9dfecbc.jpg)
both are writing.. there's no way to see who wrote what

**Comment by AndyScherzinger** [comment]
oh this loooks indeed very strange! you should have a toolbar at the top with a back arrow and also icons for starting voice/video call and a 3 dot menu...

Do you see a toolbar in other views?

**Comment by AndyScherzinger** [comment]
Also which version is shown in the settings view at the bottom?
What device is this with which Android version?

**Comment by theoneandonly-vector** [comment]
the screenshot is weird I actually have the toolbar and 3dot menu. but the text is always like this (in groups it shows the name & picture).
it's v12.1.2 on Oneplus 8pro with Android 11.

**Comment by theoneandonly-vector** [comment]
![Screenshot_20210814-074103](https://user-images.githubusercontent.com/10380051/129435376-bd7acf37-b213-4bbc-9fe3-8d0b5d857886.jpg)

**Comment by timkrueger** [comment]
@theoneandonly-vector Thanks for reporting that issue. I tried to reproduce this issue without success. 

Is it possible that you create a test account on your Nextcloud installation and send a message to that new account?

**Comment by theoneandonly-vector** [comment]
yes just did this, with the same result

Tim Krueger ***@***.***> schrieb am Mo., 16. Aug. 2021, 11:55:

**Comment by timkrueger** [comment]
> yes just did this, with the same result 

Sorry. I did not express myself clearly.

Would it be possible to provide me with this test account? You can send me the credentials to t+nctalkissue1526 (at) timkrueger (dot) me

**Comment by timkrueger** [comment]
@theoneandonly-vector Thank you very much for creating a test user on your instance. I can't reproduce your issue with 12.1.2 from the Google Playstore or with the current master.

**Comment by timkrueger** [comment]
@theoneandonly-vector could resolve the issue by himself deleting spreed, the data and reinstalling the app  on the server. So this was no client issue.

**Comment by theoneandonly-vector** [comment]
this actually is a client-side issue..
I just had this same issue again.. and was able to fix it by removing talks appdata on the phone. (reinstalling actually keeps the appdata..)
